website/meta/languages.json

```diff
@@ -434,7 +434,7 @@
             ]
         },
         {
-            "code": "xx",
+            "code": "mul",
             "name": "Multi-language",
             "models": ["xx_ent_wiki_sm", "xx_sent_ud_sm"],
```

Comment:
I assume we'll rename these models too, right?

Reply:
So many "xx"! 😄 I'll go through the codebase again and rename all remaining "xx"